queries.sql

```diff
@@ -127,4 +127,60 @@ SELECT
 FROM
     animals
 GROUP BY species, date_of_birth
-HAVING date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
\ No newline at end of file
+HAVING date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
+
+
+-- Day 3
+
+-- Write queries (using JOIN) to answer the following questions:
+-- What animals belong to Melody Pond?
+
+SELECT name
+FROM animals
+JOIN owners
+ON animals.owner_id = 4;
+
+-- List of all animals that are pokemon (their type is Pokemon).
+
+SELECT animals.name
+FROM animals
+JOIN species
+ON animals.species_id = 1;
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+
+SELECT animals.name,owners.full_name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
```

Comment:
- [ ] Checking your result for the third query I realized that `Sam Smith` does not have any animal assigned to him and from the data you have inserted in your `data.sql`, You have assigned `Agumon` to `Sam Smith`, Kindly check your query again and be sure to have a correct result. The only person that does not have an animal assigned to him is `Jodie Whittaker`

Reply:
SELECT animals.name,owners.full_name
FROM owners
LEFT JOIN an
![Screenshot 2023-08-17 141211](https://github.com/addisalemseifu/vet-clinic-database/assets/65928536/68b947c9-b560-4d98-a157-76e9e135e221)
imals ON owners.id = animals.owner_id;